dockerfiles/reverse-proxy/templates/nginx.conf.erb

```diff
@@ -12,9 +12,7 @@ http {
   include       /etc/nginx/mime.types;
   default_type  application/octet-stream;
 
-  log_format main '$proxy_protocol_addr - [$time_local] '
-                  '"$request" $status $body_bytes_sent '
-                  '"$http_referer" "$http_user_agent"';
+  log_format main <%= log_format %>;
```

Comment:
Unfortunately this kind of mangles the nginx.conf file:
<img width="771" alt="Screen Shot 2020-08-06 at 1 17 25 pm" src="https://user-images.githubusercontent.com/12392541/89487595-3ed94680-d7e9-11ea-8633-c44407dbb9a1.png">

I'm hoping this isn't a big deal because I'm assuming we won't be looking at the generated nginx files too much 😅